pinot-spi/src/main/java/org/apache/pinot/spi/utils/builder/TableConfigBuilder.java

```diff
@@ -105,6 +105,7 @@ public class TableConfigBuilder {
   private SegmentPartitionConfig _segmentPartitionConfig;
   private boolean _nullHandlingEnabled;
   private boolean _columnMajorSegmentBuilderEnabled = true;
+  private boolean _skipSegmentPreprocess;
```

Comment:
(minor) Keep the order or variable declaration consistent with methods for readability